Add Amazon Aurora (#176)
scalar-labs/scalar-kubernetes: docs/SetupDatabaseForAWS.md

* Follow this [AWS official guide](https://docs.aws.amazon.com/AmazonRDS/latest/UserGuide/USER_CreateDBInstance.html) to create an AWS RDS MySQL/PostgreSQL/Oracle/SQL Server database instance.
* (Optional) Configure advanced monitoring services with the [AWS official guide](https://docs.aws.amazon.com/AmazonRDS/latest/UserGuide/CHAP_Monitoring.html), monitoring is enabled by default on AWS RDS database instances.

> ## Amazon Aurora for MySQL and PostgreSQL

Reviewer: ```suggestion
## Amazon Aurora MySQL and Amazon Aurora PostgreSQL
```
It's not `for` MySQL. Just a MySQL-compatible Amazon Aurora.

Author: Understood. Thank you for the correction. 
I have updated the section title as suggested.

### Steps

> * Follow this [AWS official guide](https://docs.aws.amazon.com/AmazonRDS/latest/AuroraUserGuide/Aurora.CreateInstance.html) to create an Amazon Aurora cluster for MySQL/PostgreSQL.

Reviewer: ```suggestion
* Follow this [AWS official guide](https://docs.aws.amazon.com/AmazonRDS/latest/AuroraUserGuide/Aurora.CreateInstance.html) to create Amazon Aurora MySQL or Amazon Aurora PostgreSQL.
```

Author: Thank you for the feedback.
I have updated the sentence as suggested.

* Follow this [AWS official guide](https://docs.aws.amazon.com/AmazonRDS/latest/UserGuide/USER_CreateDBInstance.html) to create an AWS RDS MySQL/PostgreSQL/Oracle/SQL Server database instance.
* (Optional) Configure advanced monitoring services with the [AWS official guide](https://docs.aws.amazon.com/AmazonRDS/latest/UserGuide/CHAP_Monitoring.html), monitoring is enabled by default on AWS RDS database instances.

## Amazon Aurora for MySQL/PostgreSQL

### Steps

* Follow this [AWS official guide](https://docs.aws.amazon.com/AmazonRDS/latest/AuroraUserGuide/Aurora.CreateInstance.html) to create an Amazon Aurora cluster for MySQL/PostgreSQL.

> * (Optional) Follow this [AWS official guide](https://docs.aws.amazon.com/AmazonRDS/latest/AuroraUserGuide/Aurora.Managing.Performance.html) to manage performance and scaling for Aurora DB clusters and DB instances.

Reviewer 2: Is there any specific reason for adding performance and scaling steps for Aurora?

* Follow this [AWS official guide](https://docs.aws.amazon.com/AmazonRDS/latest/UserGuide/USER_CreateDBInstance.html) to create an AWS RDS MySQL/PostgreSQL/Oracle/SQL Server database instance.
* (Optional) Configure advanced monitoring services with the [AWS official guide](https://docs.aws.amazon.com/AmazonRDS/latest/UserGuide/CHAP_Monitoring.html), monitoring is enabled by default on AWS RDS database instances.

## Amazon Aurora for MySQL/PostgreSQL

### Steps

* Follow this [AWS official guide](https://docs.aws.amazon.com/AmazonRDS/latest/AuroraUserGuide/Aurora.CreateInstance.html) to create an Amazon Aurora cluster for MySQL/PostgreSQL.
* (Optional) Follow this [AWS official guide](https://docs.aws.amazon.com/AmazonRDS/latest/AuroraUserGuide/Aurora.Managing.Performance.html) to manage performance and scaling for Aurora DB clusters and DB instances.
* (Optional) Configure `Enhanced Monitoring` for Amazon Aurora from the [AWS official guide](https://docs.aws.amazon.com/AmazonRDS/latest/AuroraUserGuide/USER_Monitoring.OS.Enabling.html), monitoring is enabled on the Amazon Aurora cluster by default.

Note:-

> * Autoscaling can be enabled in Amazon Aurora if you have created the Amazon Aurora cluster with one primary instance and at least one Aurora Replica. Read more about it in the [AWS official guide](https://docs.aws.amazon.com/AmazonRDS/latest/AuroraUserGuide/Aurora.Integrating.AutoScaling.html).

Author: Thanks for the suggestion.
I have removed the section as it does not seem to inform anything specific related to Scalar DB/DL.
